### ISSUE TITLE
Structure of train_seq file

### ISSUE THREAD

**Issue by mortezagolzan** [Oichii/DeepPulse-pytorch#1]
Hi,
Would you please provide a sample of "train_seq.txt" ?!

**Comment by Oichii** [comment]
Assuming you have folder structure such as: 
```
  +--- Database 
  |    +--- 01-01`
  |    |    +--- frame_001.png, frame_002.png, ....
  |    +--- 02-02
  |    +--- ... 
```
where database folder contain subfolders of separate sequences split into frames. 

`train_seq.txt` file should look like: 

```
01-01
02-02
....
```

which means folder names separated by new line (one folder name per line), not containing new line at the end of file. Specific folder names depends on your folder structure.

**Comment by mortezagolzan** [comment]
Thank you for your response. 

Did you train on PhysNet? NegPearson loss makes the loss value to be upper than 1 or below this value. So my loss fluctuates between 0.9 and 1.1 and I think it does not train my network! Even when I use the absolute function of NegPearson it just gets stuck on 0.7 and does not decrease anymore. Do you have any idea? Moreover, do you have a pre-trained model of PhysNet?!

**Comment by Oichii** [comment]
Yes, I did, but I think I used MSE loss function with SGD optimizer.  
To get exact info on NegPearson loss refer to the PhysNet article (https://arxiv.org/pdf/1905.02419.pdf). 
You are right values close to 1 mean that your net is not learning, you can try different loss functions or learning rates.

Unfortunately, i don't have trained models that I can share.

**Comment by Mousumi700** [comment]
Hello  Sir,

I found your code to be very useful and I am trying to reproduce the same results. However, I am not getting the desired results and also, the loss value remains to be almost the same for subsequent epochos (implies: the network is not learning). 
I have used the same code, the only difference is the data generation format. Could you please provide me the code for creating the data format that you have created? That will be very helpful.

Thanks in advance.

**Comment by Oichii** [comment]
Each frame of the sequence is cropped to the subject's face using this script: 
[https://github.com/Oichii/DeepPulse-pytorch/blob/master/utils/face_crop.py](url)
to limit background impact on predictions.
